apps/backend/src/mcp/tools/device_management.py

```
@@ -129,7 +129,7 @@ async def add_device(
                         "Use other MCP tools to manage containers, check system metrics, etc.",
                         "Update device configuration via update_device tool if needed",
                     ],
-                    "timestamp": datetime.now(datetime.UTC).isoformat(),
+                    "timestamp": datetime.now(timezone.utc).isoformat(),
```

Ruff (0.12.2)

Undefined name `timezone`

(F821)